src/components/complex/Proposal/index.jsx

```diff
@@ -155,14 +162,22 @@ const Proposal = ({
   effectiveId,
   formattedCloseDate,
   formattedPercentageNay,
+  totalFormattedPercentageNay,
   formattedPercentageYea,
+  totalFormattedPercentageYea,
   formattedVote,
+  votingPnt,
   formattedVotingPnt,
+  votingPntDouble,
+  formattedVotingPntDouble,
   id,
   idText,
   open,
+  totalOpen,
```

Comment:
Isn't it better to maintain the same props and reflect changes into the them? the open prop here may be adjusted in such a way it becomes the totalOpen. In a logical sense I don't see why we need 2 variables. The jsx file should build the graphical interface only, it's not intended to do calculation.

Reply:
Same applies for totalPassed and quorumReached

Reply:
right